client/src/codegen/go-client.ts

```diff
@@ -705,16 +723,23 @@ type Client struct {
 }
 
 type Options struct {
-	Endpoint string
```

Comment:
Inconsistent indentation, mixing tabs and spaces. Before the change, all the code in the `static` string was indented with tabs.